base-utils/bzip2/autobuild/prepare

```diff
@@ -1,4 +1,11 @@
-sed -e 's/^CFLAGS=\(.*\)$/CFLAGS=\1 \$(BIGFILES)/' -i ./Makefile-libbz2_so
+abinfo 'Patching the CFLAGS in Makefile of libbz2.so...'
+sed -e 's/^CFLAGS=\(.*\)$/CFLAGS=\1 \$(BIGFILES)/g' \
+    -i "$SRCDIR"/Makefile-libbz2_so
 
-sed -i "s|-O2|${CFLAGS}|g" Makefile
-sed -i "s|-O2|${CFLAGS}|g" Makefile-libbz2_so
+abinfo 'Replacing CFLAGS in Makefiles with what Autobuild3 defined...'
+sed -e "s|-O2|${CFLAGS}|g" -i "$SRCDIR"/Makefile
+sed -e "s|-O2|${CFLAGS}|g" -i "$SRCDIR"/Makefile-libbz2_so
+
+abinfo 'Copying bzip.pc to SRCDIR and patching the version in it...'
+cp -v "$SRCDIR"/../bzip2.pc "$SRCDIR"
+sed -e "s|1.0.6|$PKGVER|" -i "$SRCDIR"/bzip2.pc
```

Comment:
Please do not do this. This operation is very fragile and will break any time since you downloaded the `pkgconfig` file from the Fedora `rawhide` branch.

My recommendation is to either bundle the file ourselves or use a [HERE document](https://tldp.org/LDP/abs/html/here-docs.html) construct to generate a `pkgconfig` file.

Reply:
I use HERE document construct to generate the `pkgconfig` file, thanks ~